homeassistant/components/zha/core/device.py

```diff
@@ -561,7 +561,15 @@ async def async_add_to_group(self, group_id):
 
     async def async_remove_from_group(self, group_id):
         """Remove this device from the provided zigbee group."""
-        await self._zigpy_device.remove_from_group(group_id)
+        try:
+            await self._zigpy_device.remove_from_group(group_id)
+        except (zigpy.exceptions.DeliveryError, asyncio.TimeoutError) as ex:
```

Comment:
I'm on the fence whether we should use `zigpy.exceptions.ZigbeeException` instead of `DeliveryError` here.  DeliverError is raised when the delivery fails (doh) but if usb port is disconnected or `bellows` is restarting it is going to be a different exception and either way any of those exceptions means the  device hasn't received the request.